manifests/validate.pp

```diff
@@ -7,17 +7,35 @@
 # This is used because after returning success,
 # the service is still not quite ready.
 #
+# We can validate connection either with
+#   1) root password if given or
+#   2) status password if status_check is true
+# 
 class galera::validate(
   String $action,
   String $catch,
   Integer $delay,
-  String $host = $galera::status_host,
```

Comment:
I'd rather _not_ remove these class parameters, because this makes this a breaking change for no reason.
Instead, we may declare them as OPTIONAL parameters.

Reply:
yes, agree